content-src/asrouter/templates/SimpleBelowSearchSnippet/_SimpleBelowSearchSnippet.scss

```diff
@@ -0,0 +1,51 @@
+.SimpleBelowSearchSnippet {
+  background-color: inherit;
+  border: 0;
+  box-shadow: none;
+  position: relative;
+  z-index: auto;
+
+  &.active {
+    background: var(--newtab-element-hover-color);
+    border-radius: 4px;
+  }
+
+  .innerWrapper {
+    flex-direction: column;
+    padding: 15px;
+    text-align: center;
+    width: 100%;
+
+    @media (min-width: $break-point-medium) {
```

Comment:
this looks pretty weird with DiscoveryStream enabled right now, I think we should limit this to non-ds right now; do you mind changing that and adding a comment?